lib/ddtrace/contrib/resque/resque_job.rb

```diff
@@ -0,0 +1,30 @@
+require 'ddtrace/ext/app_types'
+
+module Datadog
+  module Contrib
+    # Uses Resque job hooks to create traces
+    module ResqueJob
+      SERVICE = 'resque'.freeze
+
+      def self.extended(base)
+        Datadog::Pin.new(SERVICE, app_type: Ext::AppTypes::WORKER).onto(base)
+      end
+
+      # rubocop:disable Style/RedundantSelf
```

Comment:
why can't we use simply follow rubocop suggestion?

Reply:
I initially thought that was needed to call for the base class. I've changed it!